### ISSUE TITLE
No longer works, please fix

### ISSUE THREAD

**Issue by Emusta** [Jayapraveen/INE-courses-downloader#12]
Warning! Until the script completes execution do not access INE website or mobile app
as it might invalidate the session!

Cloud
Data Science
Development
IT Essentials
Networking
Cyber Security
Choose Method Of Operation:
1.Site Rip
2.Select Individual Course

Choose Preferred Video Quality
1.1080p
2.720p
2
Choose Method Of Selecting Course
1.Enter url
2.Choose from the above listed course
2
Please enter the number corresponding to the course you would like to download
127
Traceback (most recent call last):
File "Ine.py", line 461, in
if(course["access"]["related_passes"][0]["name"] in access_pass):
TypeError: argument of type 'NoneType' is not iterable

**Comment by Jayapraveen** [comment]
Fixed in the commit `b29c48`.

**Comment by MatterSec** [comment]
> Fixed in the commit `b29c48`.
Just downloaded latest commit, but getting the follow error now.

```
835. CCIE R&S Advanced Technologies Class Version 4.5
Total 836 courses

Choose Preferred Video Quality
1.1080p
2.720p
1
Choose Method Of Selecting Course
1.Enter url
2.Choose from the above listed course
2
Please enter the number corresponding to the course you would like to download
835

Traceback (most recent call last):
  File "INE-Downloader.py", line 456, in <module>
    if(course["access"]["related_passes"][4]["name"] in access_pass):
IndexError: list index out of range
```


Thanks

**Comment by Jayapraveen** [comment]
> > Fixed in the commit `b29c48`.
> > Just downloaded latest commit, but getting the follow error now.
> 
> ```
> 835. CCIE R&S Advanced Technologies Class Version 4.5
> Total 836 courses
> 
> Choose Preferred Video Quality
> 1.1080p
> 2.720p
> 1
> Choose Method Of Selecting Course
> 1.Enter url
> 2.Choose from the above listed course
> 2
> Please enter the number corresponding to the course you would like to download
> 835
> 
> Traceback (most recent call last):
>   File "INE-Downloader.py", line 456, in <module>
>     if(course["access"]["related_passes"][4]["name"] in access_pass):
> IndexError: list index out of range
> ```
> 
> Thanks

Fixed in `6ea35ca`.
